src/app/modules/item/helpers/permissions-edit-dialog-texts.ts

```diff
@@ -1,6 +1,5 @@
-import { ProgressSelectValue } from
```

Comment:
Small issue about the filename: it's a bit misleading because it's not about a dialog only anymore :grimacing: 
I guess something simpler like `permissions-edit-texts` or even `permissions-texts` would be more appropriate.